plugins/modules/swim_intent.py

```diff
@@ -965,15 +979,15 @@ def get_diff_tagging(self):
                 deviceFamilyIdentifier=self.have.get("device_family_identifier"),
                 deviceRole=tagging_details.get("device_role")
             )
-            self.log("Image params for tagging image as golden:" + str(image_params))
+            self.log("Image params for tagging image as golden: {0}".format(str(image_params)), "INFO")
```

Comment:
self.log("Parameters for tagging the image as golden: {0}".format(str(image_params)), "INFO")

Reply:
added.